README.md

```diff
@@ -193,24 +221,52 @@ The following instructions assume a Unix-like shell (e.g. bash).
       go env GOROOT GOPATH
 
 - Ensure `$GOPATH/bin` is on your `$PATH`.
+
+- Clone the dcrdata repository. It **must** be cloned into the following directory.
+
+      git clone https://github.com/decred/dcrdata $GOPATH/src/github.com/decred/dcrdata
+
+Once Golang is installed and the dcrdata project is cloned proceed to the build steps.
+
+### Build from source for golang >= 1.11
+
+Golang 1.11 introduced a new [dependency management feature](https://github.com/golang/go/wiki/Modules) that
+negates the use of third party tooling such as `dep`. Using `Go mod` will be the dependency managment tool of choice
+for dcrdata and other Decred projects going forward.
+
+Usage is simple and nothing is required except Golang 1.11.
+
+1. `go build`
+
+If you want to pre staging dependencies without building you can run
+
+1. `go mod download`
+
+Golang's builtin dependency management resolution will start processing
+the source code and downloading dependencies on the fly and then update the `go.mod` and `go.sum` files automatically.
+
+### Build from source for golang < 1.11
+
+Building with the `dep` dependency management tool.
```

Comment:
There are really two options with go 1.10.  The first is `dep`, as described here, and the other is `vgo`.  The go 1.11 way and and the vgo way are supported and described here: https://github.com/golang/go/wiki/Modules#installing-and-activating-module-support

It's desirable to have builds from the same dependency tree regardless of go version, so I'd advise the vgo way if the user is unable to use go 1.11.  We've not kept the dep files updated to match the module defs, and it might not be possible.